notebooks/cookbook/openmm_8_tutorial_rest_draft_v2.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "73344755",
+   "metadata": {},
+   "source": [
+    "# Run a REST simulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e110b4e9",
+   "metadata": {},
+   "source": [
+    "Replica exchange solute tempering (REST) is an enhanced sampling approach that aims to improve sampling for a user-defined subset of atoms (i.e., REST atoms) by increasing its effective temperature.\n",
+    "\n",
+    "REST involves running replica exchange to sample a set of thermodynamic states, where one state contains the unmodified Hamiltonian and all other states involve modified Hamiltonians where REST atom interactions are scaled to increased temperatures (Wang et al. 2011, DOI: 10.1021/jp204407d).\n",
+    "\n",
+    "In this tutorial, we show how to build an REST-capable OpenMM system. At the end, we show how to use OpenMMTools (https://github.com/choderalab/openmmtools) objects to run REST."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3600abfc",
+   "metadata": {},
+   "source": [
+    "## 01 - Construct a vanilla OpenMM system"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fc22b65",
+   "metadata": {},
+   "source": [
+    "We first load a PDB file consisting of villin in water and build a `System` from it, which we refer to as the \"vanilla system\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b1a7f398",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openmm\n",
+    "from openmm import app, unit"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ba6c3c6c",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pdb = app.PDBFile('/home/zhangi/choderalab/openmm-cookbook/notebooks/cookbook/villin.pdb')\n",
```

Comment:
Absolute paths won't work on users' computers.  Can you use a relative path instead, and provide a link for the user to download the file (https://raw.githubusercontent.com/openmm/openmm-cookbook/main/notebooks/cookbook/villin.pdb)?